windows/win_msg.ps1

```diff
@@ -0,0 +1,52 @@
+#!powershell
+# This file is part of Ansible
+#
+# Copyright 2015, Jon Hawkesworth (@jhawkesworth) <figs@unity.demon.co.uk>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+$params = Parse-Args $args
+$result = New-Object PSObject
+Set-Attr $result "changed" $False
+$to = Get-Attr -obj $params -name to -default "*" -failifempty $False -resultobj $result
+$display_seconds = Get-Attr -obj $params -name display_seconds -default "10" -failifempty $False -resultobj $result
+$wait = Get-Attr -obj $params -name wait -default $False -failifempty $False -resultobj $result
+$msg = Get-Attr -obj $params -name msg -failifempty $True -resultobj $result
+
+$msg_args = @($to, "/TIME:$display_seconds")
+
+If ($wait) 
+{
+  $msg_args += "/W"
```

Comment:
The Microsoft documentation seems to be incorrect on this, when using the /W argument it waits until the /TIME specified is over before displaying the message for that same time period, e.g. when specifying /TIME:10 it will wait 10 seconds before showing the message and then wait another 10 seconds before existing the call.

EDIT: After looking further Microsoft is correct with this but it sends message in a serial fashion, so if there are multiple users logged on it will send to 1 and wait the time allotted and then send to another user.